components/calendar/src/cal/chinese.rs

```diff
@@ -328,7 +328,7 @@ impl<A: AsCalendar<Calendar = Chinese>> Date<A> {
     }
 }
 
-type ChineseCB = calendrical_calculations::chinese_based::Chinese;
+type ChineseCB = chinese_based::Chinese;
```

Comment:
nit: this was fully qualified on purpose because there's also a local `chinese_based` module